framework/db/Command.php

```diff
@@ -408,14 +408,27 @@ public function queryColumn()
      *
      * @param string $table the table that new rows will be inserted into.
      * @param array $columns the column data (name => value) to be inserted into the table.
+     * @param array $returnColumns the column names which values to be returned after inserting rows.
+     * @param array $returnParams if not empty after calling this method, each value is bound to the query.
      * @return Command the command object itself
+     * @throws NotSupportedException if $returnColumns is set but the underlying DBMS doesn't support returning
      */
-    public function insert($table, $columns)
+    public function insert($table, $columns, $returnColumns = null, &$returnParams = null)
```

Comment:
Yes. This way we'll keep current signature so it will be backwards compatible and could be applied to 2.0.x instead of 2.1.x.